components/x-audio/src/components/PlaybackRate.jsx

```diff
@@ -0,0 +1,27 @@
+import { h } from '@financial-times/x-engine';
+import * as PropTypes from 'prop-types';
+import classNameMap from './classnames-helper';
+
+import { getNextPlaybackRate } from './playback-rates';
+
+export const PlaybackRate = ({
+	onClick,
+	rate
+}) => (
+	<button
+		aria-label="Change playback speed"
```

Comment:
I found the best solution here was to update the aria-label to inform the user what the speed would change to. `aria-live` didn't make a difference. The user can get the button label to repeat

Reply:
thanks @debugwand, I've added this now